File: week01-210619-210625/moregorenine/baekjoon/step2_if/q2_9498_test_score/main.py
import sys

number = int(sys.stdin.readline())


def printScoreToGrades(number):
    grades = ""
    if number >= 90:
        grades = "A"
    elif number >= 80:
        grades = "B"
    elif number >= 70:
        grades = "C"
    elif number >= 60:
        grades = "D"
    else:
        grades = "F"
    print(grades)


printScoreToGrades(number)

File: week01-210619-210625/moregorenine/baekjoon/step2_if/q2_9498_test_score/Main.java
package step2_if.q2_9498_test_score;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int number = Integer.parseInt(st.nextToken());

        printScoreToGrades(number);

    }

    private static void printScoreToGrades(int number) {
        String grades = "";
        if (number >= 90) {
            grades = "A";
        } else if (number >= 80) {
            grades = "B";
        } else if (number >= 70) {
            grades = "C";
        } else if (number >= 60) {
            grades = "D";
        } else {
            grades = "F";
        }
        System.out.println(grades);
    }

}
